.github/workflows/build.yml

```diff
@@ -33,3 +33,5 @@ jobs:
       - name: Build Centos 8 RPM
         run: docker run -v `pwd`:/scylla-machine-image -w /scylla-machine-image  --rm centos:8 bash -c './dist/redhat/build_rpm.sh -t centos8'
 
+      - name: Build Ubuntu DEB
+        run: docker run -v `pwd`:/scylla-machine-image -w /scylla-machine-image  --rm fedora:33 bash -c './dist/debian/build_deb.sh'
```

Comment:
why do you build Debian package using Fedora container?

Reply:
It's aligned with scylla main repo (relocatable package based packaging), we run all packaging on Fedora33 on main repo.
We haven't enable to generate relocatable package for scylla-machine-image, but probably it's better to align the way to build the package between projects.

Reply:
@avikivity do we need to enable support relocatable package for scylla-machine-image too? how about to include machine-image in unified tar.gz?

Reply:
@syuu1228 no need, at least for now.

machine-image has its own release stream (it's not a submodule of scylla.git) so it can't be part of unified.tar.gz. Eventually it will receive its own versioning - no need to release machine-image-4.4.3 when we release scylla-4.4.3, it can have a separate lifecycle like scylla-monitoring.

Reply:
So we need to build Debian package on Debian/Ubuntu container.

Reply:
Supported both Fedora33/Ubuntu20.04.